js/src/forum/extenders/extendDiscussionSearchSource.tsx

```diff
@@ -3,7 +3,7 @@ import { extend } from 'flarum/common/extend';
 import DiscussionsSearchSource from 'flarum/forum/components/DiscussionsSearchSource';
 
 export default function extendDiscussionsSearchSource() {
-  extend(DiscussionsSearchSource.prototype, 'queryMutators', function (mutators: string[]) {
+  extend(DiscussionsSearchSource.prototype, 'queryMutators', function (mutators) {
```

Comment:
```suggestion
  @ts-ignore
  extend(DiscussionsSearchSource.prototype, 'queryMutators', function (mutators: string[]) {
```

Let's still cast it as `string[]` like it was before so `mutators.push('-is:solved');`  doesn't error and for now add a `@ts-ignore`. Seems like a slight change in core would be needed